bciers/apps/reporting/src/data/jsonSchema/nonAttributableEmissions/nonAttributableEmissions.ts

```diff
@@ -0,0 +1,107 @@
+import NestedArrayFieldTemplate from "@bciers/components/form/fields/NestedArrayFieldTemplate";
+
+export const nonAttributableEmissionSchema: RJSFSchema = {
+  type: "object",
+  title: "Non-Attributable Emissions",
+  properties: {
+    info_note: { type: "object", readOnly: true },
+    emissions_exceeded: {
+      type: "boolean",
+      title: "Did your non-attributable emissions exceed 100 tCO2e?",
+    },
+  },
+};
+
+export const nonAttributableEmissionUiSchema = {
+  "ui:FieldTemplate": FieldTemplate,
+  "ui:classNames": "form-heading-label",
+  "ui:order": ["info_note", "emissions_exceeded", "activities"],
+  info_note: {
+    "ui:FieldTemplate": TitleOnlyFieldTemplate,
+    "ui:title": NonAttributableEmmissionsInfo,
+  },
+  emissions_exceeded: { "ui:widget": RadioWidget },
+  activities: {
+    "ui:ArrayFieldTemplate": NestedArrayFieldTemplate,
+
+    "ui:FieldTemplate": FieldTemplate,
+    "ui:options": {
+      arrayAddLabel: "Add Activity",
+      padding: "p-2",
+      label: false,
+      bgColor: "white",
+      showHr: true,
+    },
+    items: {
+      "ui:order": ["activity", "source_type", "emission_category", "gas_type"],
+      emission_category: {
+        "ui:widget": SelectWidget,
+        "ui:placeholder": "Select Emissions Category",
+        "ui:options": { style: { width: "100%", textAlign: "justify" } },
+      },
+      gas_type: {
+        "ui:widget": CheckboxGroupWidget,
+        "ui:options": {
+          alignment: "top",
+          columns: 3,
+        },
+      },
+    },
+  },
+};
+
+export const generateUpdatedSchema = (
+  gasTypes: { id: number; chemical_formula: string }[],
+  emissionCategories: { id: number; category_name: string }[],
+): RJSFSchema => ({
+  ...nonAttributableEmissionSchema,
+  dependencies: {
+    emissions_exceeded: {
+      oneOf: [
+        {
+          properties: { emissions_exceeded: { enum: [false] } },
+        },
+        {
+          properties: {
+            emissions_exceeded: { enum: [true] },
+            activities: {
+              type: "array",
+              items: {
+                type: "object",
+                properties: {
+                  activity: { type: "string", title: "Activity Name" },
+                  source_type: { type: "string", title: "Source Type" },
+                  gas_type: {
+                    type: "array", // Adjusted to array type
+                    title: "Gas Type (Optional)",
+                    items: {
+                      type: "string",
+                      enum: gasTypes.map((gas) => gas.chemical_formula),
+                    },
+                    uniqueItems: true,
+                  },
+                  emission_category: {
+                    type: "string",
+                    title: "Emission Category (Optional)",
+                    enum: emissionCategories.map(
+                      (category) => category.category_name,
```

Comment:
I seem to remember someone mentioning that the Gas Type & Emission Category fields on this page are actually not optional, but I can't find any figma comment or mention of it in the card. Could you confirm with @zoeyli-46 to find out whether or not these fields should be optional?